### PR TITLE
Add Elixir integrations

### PR DESCRIPTION
**Pull request by cairoeth**
Adding Elixir integrations for TVL

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/elixir exports TVL: 
        
 
 
    ethereum                  257.01 M
    ethereum-pool2            95.68 M
    pool2                     95.68 M
    arbitrum                  10.36 M
    
    total                    267.37 M

File: projects/elixir/index.js [hunk 1]
```diff
@@ -1,27 +1,74 @@
...
+    tvl: async (api) => {
+      const deusdSupply = await api.call({ target: deUSD, abi: "erc20:totalSupply" })
+      api.add(deUSD, deusdSupply);
+      await api.sumTokens({ owners: [COMMITS, FOUNDATION, FOUNDATION_2], tokens: [ADDRESSES.ethereum.STETH, ADDRESSES.null, ADDRESSES.ethereum.SDAI] })
+    },
+    pool2: sumTokensExport({ owner: DEUSD_LP_STAKING, tokens: LP_TOKENS })
   },
+  arbitrum: {
+    tvl: async (api) => {
+      await integration(api, VERTEX_MANAGER, [1, 2, 3, 4, 5, 6, 8, 10, 12, 14, 16, 18, 20, 22, 24, 26, 28, 30, 31, 34, 36, 38, 40, 41, 44, 46, 48, 50, 52, 54, 56, 58, 60, 62], [
```
Comment (g1nt0ki) [inline projects/elixir/index.js:60]: hi @cairoeth what is this/why is it counted as tvl?

**Comment by llamatester** [comment]
The adapter at projects/elixir exports TVL: 
        
 
 
    ethereum                  259.14 M
    ethereum-pool2            95.65 M
    pool2                     95.65 M
    arbitrum                  9.84 M
    
    total                    268.98 M

File: projects/elixir/index.js (hunk 1 above)
Comment (cairoeth) [reply projects/elixir/index.js:48]: these are the elixir orderbook integrations: liquidity that is deposited to the elixir contracts and then used to market make on the exchanges integrated.

Comment (g1nt0ki) [reply projects/elixir/index.js:48]: are the backing of these tokens already counted towards deUSD supply?

Comment (cairoeth) [reply projects/elixir/index.js:48]: no, they are different products and their liquidity is not connected.

**Pull request merged**